server/model/remandReviewModel.ts

```diff
@@ -61,16 +77,29 @@ export default class RemandReviewModel {
             text: 'Offences',
           },
           value: {
-            html: `<ul class="govuk-list govuk-list--bullet">
+            html: `<div class="govuk-list">
```

Comment:
Should this be a div with list class? I'm not sure thats how the govuk-list style should be used.

Reply:
I've removed the classes as they are no longer used.